targets/mgyamlfmt/targets.go

```diff
@@ -0,0 +1,55 @@
+package mgyamlfmt
+
+import (
+	"bytes"
+	"io/fs"
+	"io/ioutil"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"go.einride.tech/mage-tools/mglog"
+	"go.einride.tech/mage-tools/mgtool"
+	"gopkg.in/yaml.v3"
+)
+
+func FormatYaml() error {
+	logger := mglog.Logger("format-yaml")
+	logger.Info("formatting yaml files...")
+	return filepath.Walk(mgtool.GetGitRootPath("."), func(path string, _ fs.FileInfo, err error) error {
```

Comment:
btw I think [`filepath.WalkDir`](https://pkg.go.dev/path/filepath#WalkDir) is the recommended func nowadays due to it now having to call `stat` on _every_ file. No need to change, just a heads-up

Reply:
Fixed